.cirrus.yml

```diff
@@ -0,0 +1,16 @@
+freebsd_instance:
+  image: freebsd-12-0-release-amd64
+freebsd_test_task:
+  env:
+    CC: "clang70"
+    CXX: "clang++70"
+  install_script:
+    - pkg install -y cmake gmake llvm70
+  build_script:
+    - mkdir build
+    - cd build
+    - cmake .. -DCMAKE_BUILD_TYPE=Release
+    - gmake -j$(sysctl hw.ncpu | cut -d ' ' -f 2) install
```

Comment:
[`sysctl -n hw.ncpu`](https://www.freebsd.org/cgi/man.cgi?sysctl)

then you won't need to cut anything

Reply:
Nice, I didn't know about `-n`. Thanks!